src/networking.c

```diff
@@ -809,12 +810,21 @@ void addReplyLongLong(client *c, long long ll) {
     else if (ll == 1)
         addReply(c,shared.cone);
     else
-        addReplyLongLongWithPrefix(c,ll,':');
+        addReplyLongLongWithPrefix(c,ll,':',0);
+}
+
+void addReplyUnsignedLongLong(client *c, unsigned long long ll) {
+    if (ll == 0)
+        addReply(c,shared.czero);
+    else if (ll == 1)
+        addReply(c,shared.cone);
+    else
+        addReplyLongLongWithPrefix(c,ll,':',1);
```

Comment:
RESP3 says:
```
Number: an integer in the signed 64 bit range
```

Reply:
@oranagra does that mean that RESP3 does not support anything bigger than LLONG_MAX as a "number"?

Reply:
yes, that's how it looks like in the spec.
of course since that's just a string, you can put bigger values, but not sure what the client will do with that.